<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtList</class>
 <widget class="QMainWindow" name="QtList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtList</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QToolBox" name="toolBox">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>1000</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="lstOp">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>300</width>
          <height>460</height>
         </rect>
        </property>
        <attribute name="label">
         <string>列表操作</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QToolButton" name="tbtnListInit">
           <property name="text">
            <string>tbtnListInit</string>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QToolButton" name="tbtnListClear">
           <property name="text">
            <string>tbtnListClear</string>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QToolButton" name="tbtnListInsert">
           <property name="text">
            <string>tbtnListInsert</string>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QToolButton" name="tbtnListAppend">
           <property name="text">
            <string>tbtnListAppend</string>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QToolButton" name="tbtnListDelete">
           <property name="text">
            <string>tbtnListDelete</string>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="lstSort">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>300</width>
          <height>460</height>
         </rect>
        </property>
        <attribute name="label">
         <string>列表排序</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QCheckBox" name="chkSort">
           <property name="text">
            <string>可排序</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QToolButton" name="tbtnSortAsc">
           <property name="text">
            <string>升序</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/icon_form/a-Icon_jiansuoliebiao_paixujiantoushang.svg</normaloff>:/icon_form/a-Icon_jiansuoliebiao_paixujiantoushang.svg</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QToolButton" name="tbtnSortDes">
           <property name="text">
            <string>降序</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/icon_form/a-Icon_jiansuoliebiao_paixujiantouxia.svg</normaloff>:/icon_form/a-Icon_jiansuoliebiao_paixujiantouxia.svg</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>326</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="lstSign">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>300</width>
          <height>460</height>
         </rect>
        </property>
        <attribute name="label">
         <string>信号发射时机</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QToolButton" name="tbtnClearText">
             <property name="text">
              <string>清空文本</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tbtnAddLine">
             <property name="text">
              <string>添加空行</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QPlainTextEdit" name="txtInfo"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="0">
         <widget class="QListWidget" name="listWidget">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <item>
           <property name="text">
            <string>item1</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/icon_form/caozuo-wendu.svg</normaloff>:/icon_form/caozuo-wendu.svg</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>item2</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/icon_form/caozuo_gaosu.svg</normaloff>:/icon_form/caozuo_gaosu.svg</iconset>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QToolButton" name="tbtnSelectItem">
            <property name="text">
             <string>选择菜单</string>
            </property>
            <property name="popupMode">
             <enum>QToolButton::MenuButtonPopup</enum>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tbtnSelectAll">
            <property name="text">
             <string>tbtnSelectAll</string>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tbtnSelectNone">
            <property name="text">
             <string>tbtnSelectNone</string>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tbtnSelectInvs">
            <property name="text">
             <string>tbtnSelectInvs</string>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>当前项变化</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtCurrItem"/>
          </item>
          <item>
           <widget class="QCheckBox" name="chkItemEditable">
            <property name="text">
             <string>项可编辑</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionInitList"/>
   <addaction name="actionClearList"/>
   <addaction name="actionInsertItem"/>
   <addaction name="actionAddItem"/>
   <addaction name="actionDelItem"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionInitList">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_form/liebiao_.svg</normaloff>:/icon_form/liebiao_.svg</iconset>
   </property>
   <property name="text">
    <string>InitList</string>
   </property>
   <property name="toolTip">
    <string>InitList</string>
   </property>
  </action>
  <action name="actionClearList">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_form/qingkong.svg</normaloff>:/icon_form/qingkong.svg</iconset>
   </property>
   <property name="text">
    <string>ClearList</string>
   </property>
   <property name="toolTip">
    <string>ClearList</string>
   </property>
  </action>
  <action name="actionInsertItem">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_form/xinzengcaidan.svg</normaloff>:/icon_form/xinzengcaidan.svg</iconset>
   </property>
   <property name="text">
    <string>InsertItem</string>
   </property>
   <property name="toolTip">
    <string>InsertItem</string>
   </property>
  </action>
  <action name="actionAddItem">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_form/xinzeng.svg</normaloff>:/icon_form/xinzeng.svg</iconset>
   </property>
   <property name="text">
    <string>AddItem</string>
   </property>
   <property name="toolTip">
    <string>AddItem</string>
   </property>
  </action>
  <action name="actionDelItem">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_form/shanchu_2.svg</normaloff>:/icon_form/shanchu_2.svg</iconset>
   </property>
   <property name="text">
    <string>DelItem</string>
   </property>
   <property name="toolTip">
    <string>DelItem</string>
   </property>
  </action>
  <action name="actionSelectItem">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_form/liebiao_.svg</normaloff>:/icon_form/liebiao_.svg</iconset>
   </property>
   <property name="text">
    <string>SelectItem</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_form/liebiao.svg</normaloff>:/icon_form/liebiao.svg</iconset>
   </property>
   <property name="text">
    <string>SelectAll</string>
   </property>
  </action>
  <action name="actionSelectNone">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_form/empty.svg</normaloff>:/icon_form/empty.svg</iconset>
   </property>
   <property name="text">
    <string>SelectNone</string>
   </property>
  </action>
  <action name="actionSelectInvs">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_form/caozuo_gaosu.svg</normaloff>:/icon_form/caozuo_gaosu.svg</iconset>
   </property>
   <property name="text">
    <string>SelectInvs</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_form/caozuo_qipao.svg</normaloff>:/icon_form/caozuo_qipao.svg</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
